fbrp/src/fbrp/__init__.py

```diff
@@ -6,6 +6,7 @@
 from importlib.machinery import SourceFileLoader
 import click
 import os
+import sys
```

Comment:
Extraneous except for the commented code, I think?

```suggestion
```

Reply:
Done